hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/BaseSparkCommitActionExecutor.java

```diff
@@ -419,4 +427,23 @@ public Partitioner getLayoutPartitioner(WorkloadProfile profile, String layoutPa
   protected void runPrecommitValidators(HoodieWriteMetadata<HoodieData<WriteStatus>> writeMetadata) {
     SparkValidatorUtils.runValidators(config, writeMetadata, context, table, instantTime);
   }
+
+  private long dynamicSampleRecordSize(JavaRDD<HoodieRecord<T>> inputRecords) {
+    long dynamicSampleRecordSize = 1024;
```

Comment:
can we fetch this directly from the Config's default value rather than hard coding it here

Reply:
got it, i will update this code

Reply:
@nsivabalan  thanks for review, i hava update this hard coding